src/input/pipewire.c

```diff
@@ -33,7 +33,7 @@ static void on_process(void *userdata) {
     if ((samples = buf->datas[0].data) == NULL)
         return;
 
-    n_samples = buf->datas[0].chunk->size / (data->cava_audio->format / 8);
+    n_samples = buf->datas[0].chunk->size / sizeof(float);
 
```

Comment:
Again, looks like these changes I also didn't notice during refactoring. Possible need to rollback once PR is approved to merge